panels/system.py

```diff
@@ -53,25 +57,211 @@ def initialize(self, panel_name):
         info.add(self.labels['klipper_version'])
         info.add(self.labels['ks_version'])
 
+        scroll = Gtk.ScrolledWindow()
+        scroll.set_property("overlay-scrolling", False)
+        scroll.set_vexpand(True)
+
+        infogrid = Gtk.Grid()
+        infogrid.get_style_context().add_class("system-program-grid")
+        update_resp = self._screen.apiclient.send_request("machine/update/status")
+        self.update_status = False
+
+        if update_resp == False:
+            logging.info("No update manager configured")
+        else:
+            self.update_status = update_resp['result']
+            vi = update_resp['result']['version_info']
+            items = sorted(list(vi))
+            i = 0
+            for prog in items:
+                self.labels[prog] = Gtk.Label("")
+                self.labels[prog].set_hexpand(True)
+                self.labels[prog].set_halign(Gtk.Align.START)
+
+                self.labels["%s_status" % prog] = self._gtk.Button()
+                self.labels["%s_status" % prog].set_hexpand(False)
+                self.labels["%s_status" % prog].connect("clicked", self.update_program, prog)
+                self.labels["%s_box" % prog] = Gtk.Box()
+                self.labels["%s_box" % prog].set_hexpand(False)
+                self.labels["%s_info" % prog] = self._gtk.ButtonImage("information",None, None, .7, .7)
```

Comment:
can this be "info" instead of "information" just to not add another very similar icon...

Reply:
Good catch. I don't know how I missed that icon